Sources/ProjectSpec/Dependency.swift

```diff
@@ -10,6 +10,7 @@ public struct Dependency: Equatable {
     public var removeHeaders: Bool = true
     public var link: Bool?
     public var implicit: Bool = false
+    public var includeRelated: Bool?
```

Comment:
Seen as this is only relevant to carthage dependencies, could it be an associated value to the enum (defaulting to false)?

```swift
public enum DependencyType: Equatable {
    case target
    case framework
    case carthage(includeRelated: Bool)
    case sdk

    public static var carthage: DependencyType {
        return .carthage(includeRelated: false)
    }
}
```

(This works with a few minor changes; tests pass).

Reply:
great idea, will work on it.